pype/plugins/hiero/publish/extract_frames.py

```diff
@@ -18,20 +18,22 @@ class ExtractFrames(pype.api.Extractor):
 
     def process(self, instance):
         staging_dir = self.staging_dir(instance)
-        output_template = os.path.join(staging_dir, instance.data["name"])
+        output_template = (
+            os.path.join(staging_dir, instance.data["name"]) +
+            ".{:04d}.{}"
+        )
         sequence = hiero.ui.activeSequence()
         files = []
         for frame in instance.data["frames"]:
+            # Convert to uncompressed exr with oiiotool.
             track_item = sequence.trackItemAt(frame)
             media_source = track_item.source().mediaSource()
             input_path = media_source.fileinfos()[0].filename()
             input_frame = (
                 track_item.mapTimelineToSource(frame) +
                 track_item.source().mediaSource().startTime()
             )
-            format = instance.data["format"]
-            output_path = output_template
-            output_path += ".{:04d}.{}".format(int(frame), format)
+            exr_output_path = output_template.format(int(frame), "exr")
 
             args = ["oiiotool"]
```

Comment:
Hey @tokejepsen is it possible to change this to `os.getenv("PYPE_OIIO_PATH")` and set the path to the oiiotool in pype-config/environments/global.json:PYPE_OIIO_PATH ?

Reply:
Ohh, is `oiiotool` command not guaranteed?

Reply:
no, it is not. that is why we are having the env var for it. this way we are avoiding a env clashes. 
![image](https://user-images.githubusercontent.com/40640033/129723838-4e106dbb-4429-4e2e-9213-c8b49706a40c.png)